pkg/proxy/util/utils_unsupported.go

```diff
@@ -0,0 +1,43 @@
+// +build !linux
```

Comment:
I would suggest put this file in pkg/proxy/ipvs and rename it to `netlink_unsupported.go`

Reply:
The netlink specific functions can be used by other proxies and it is not associated with ipvs, as thockin says, put it in util may be a better place ?

Reply:
If other proxiers consume these functions, then they should be in util. Sorry, I can't find any now...